openapi.yaml

```diff
@@ -25,105 +32,433 @@ security:
   - OAuth2:
       - full
 paths:
-  /pets:
+  /pizzas:
     get:
-      summary: List all pets
+      description: Get all pizzas
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - pizza
       parameters:
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: name
+          description: Search pizzas by name
           required: false
           type: string
+        - in: query
+          name: ovenTemp
+          description: Search pizzas by bake temperature (in degrees Fahrenheit)
+          required: false
+          type: integer
+        - in: query
+          name: bakeTime
+          description: Search pizzas by baking time (in minutes)
+          type: integer
+        - in: query
+          name: ingredientsList
+          description: Search for pizzas with all the ingredients specified by this list of ingredient IDs
+          type: array
+          items:
+            type: string
+        - in: query
+          name: doughId
+          description: Search for pizzas with the dough matching this dough ID
+          type: string
       responses:
-        '200':
-          description: Successful response
+        "200":
+          description: A list of pizzas
           schema:
-            $ref: '#/definitions/PetsResult'
-        '400':
-          $ref: '#/responses/400'
-        '500':
-          $ref: '#/responses/500'
+            $ref: "#/definitions/PizzasResults"
+        "500":
+          $ref: "#/responses/500"
     post:
-      summary: Add a new pet
-      operationId: postPet
+      description: Create a new pizza recipe
       tags:
-        - pets
+        - pizza
       parameters:
         - in: body
-          name: body
+          name: attributes
+          description: attributes to assign to the new pizza
           required: true
           schema:
-            $ref: '#/definitions/PetPostBody'
+            $ref: "#/definitions/PizzaAttributes"
```

Comment:
Could you change the request body to be a resource object here? https://jsonapi.org/format/#crud-creating